src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -35,12 +43,161 @@
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.mob_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicMob IDs.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "mob_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getMobNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.active_mobs>
+            // @returns ListTag(MythicMobsMobTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all active MythicMobs on the server.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "active_mobs", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (ActiveMob entity : MythicMobsBridge.getMobManager().getActiveMobs()) {
+                    list.addObject(new MythicMobsMobTag(entity));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.spawners>
+            // @returns ListTag(MythicSpawnerTag)
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of all MythicSpawners.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "spawners", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (MythicSpawner spawner : MythicMobsBridge.getSpawnerManager().getSpawners()) {
+                    list.addObject(new MythicSpawnerTag(spawner));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <mythicmobs.damage_modifiers[<mob_id>]>
+            // @returns MapTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a map of the damage modifiers for a MythicMob from a Mob ID.
+            // -->
+            tagProcessor.registerTag(MapTag.class, ElementTag.class, "damage_modifiers", (attribute, object, id) -> {
+                Optional<MythicMob> optionalMythicMob = MythicBukkit.inst().getMobManager().getMythicMob(id.asString());
+                if (!optionalMythicMob.isPresent()) return null;
+                MythicMob mob = optionalMythicMob.get();
+                MapTag result = new MapTag();
+                for (Map.Entry<String, Double> entry : MythicMobsBridge.getDamageModifiers(mob).entrySet()) {
+                    result.putObject(entry.getKey(), new ElementTag(entry.getValue().toString()));
```

Comment:
`new ElementTag(entry.getValue().toString())` - the `toString` is redundant, `ElementTag` has a double constructor

Reply:
Not only redundant, actively bad at that - the ElementTag double constructor is smarter than just a toString